nightwatch/features/event-reconciliation/unreconciled-movements.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Unreconciled Movements
+
+  Background:
+    Given I am a logged in user
+    And There are no existing centres
+    And The following centres exist:
+      | name | male_capacity | female_capacity | male_cid_name  | female_cid_name |
+      | one  | 1000          | 10000           | oneman,onebman | onewoman        |
+    Given I am on the wallboard
+
+  Scenario: Unreconciled Out Movement shows as Scheduled Outgoing and does not affect Availability
+    Given The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
+    When I submit the following movements:
+      | MO In/MO Out | Location | MO Ref. | MO Date | MO Type | CID Person ID |
+      | Out          | oneman   | 111     | now     | Removal | 1             |
+    Then The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 1    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
+
+  Scenario: Unreconciled In Movement shows as Scheduled incoming and reduces availability
```

Comment:
If you want to test like that sure

On Wednesday, 4 May 2016, Chris Nesbitt-Smith <notifications@github.com>
wrote:

> In nightwatch/features/event-reconciliation/unreconciled-movements.feature
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6#discussion_r62002283>
> :
>
> > +    When I submit the following movements:
> > +      | MO In/MO Out | Location | MO Ref. | MO Date | MO Type | CID Person ID |
> > +      | Out          | oneman   | 111     | now     | Removal | 1             |
> > +    Then The Centre "one" should show the following under "Male":
> > +      | Contractual Capacity   | 1000 |
> > +      | Occupied               | 0    |
> > +      | Beds out of commission | 0    |
> > +      | Contingency            | 0    |
> > +      | Prebookings            | 0    |
> > +      | Availability           | 1000 |
> > +      | Scheduled incoming     | 0    |
> > +      | Scheduled outgoing     | 1    |
> > +      | Unexpected incoming    | 0    |
> > +      | Unexpected outgoing    | 0    |
> > +
> > +  Scenario: Unreconciled In Movement shows as Scheduled incoming and reduces availability
>
> we already have this test this in the movement.feature?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6/files/8d568934e372574dcdb57ebb6c0def7d6fba6f57#r62002283>
>